### PR TITLE
drive update for kafka

### PR DESCRIPTION
**Pull request by willb0t**
added menu item for kafka

### CODE REVIEW

**Comment by willb0t** [comment]
try one

File: templates/kafka/4/rancher-compose.yml [hunk 1]
```diff
@@ -58,31 +58,43 @@
...
+      label: "VOLUME Driver"
+      required: true
+      default: "local"
+      type: enum
+      options: # List of options if using type of `enum`
+        - local
+        - rancher-nfs
+        - rancher-efs
+        - rancher-ebs
       type: "service"
```
Comment (rawmind0) [inline templates/kafka/4/rancher-compose.yml]: This type "service" is from zk_link variable, you broke it.

File: templates/kafka/4/docker-compose.yml [hunk 2]
```diff
@@ -41,6 +41,6 @@ broker-volume:
     - SERVICE_GID=10003
     - SERVICE_VOLUME=${kafka_log_dir}
   volumes:
-    - ${kafka_log_dir}
-  volume_driver: local
+    - driver: ${kafka_log_dir}
```
Comment (rawmind0) [inline templates/kafka/4/docker-compose.yml]: This line is wrong, why did you change it?
```
-    - driver: ${kafka_log_dir}
+    - ${kafka_log_dir}
```

**Review by rawmind0 — CHANGES_REQUESTED** [review]

**Comment by rawmind0** [comment]
Hi @willb0t , thanks for the PR and contribute. 

PR has some errors and is not working. PR needs to be tested locally before commit. ;)

Please take a look to the changes requested. 

Also, build is failing. You are trying to rename `templates/zookeeper/4/docker-compose.yml.tpl` to `templates/zookeeper/4/docker-compose.yml`, why?? Could you please remove it from the PR??

**Comment by cjellick** [comment]
Very stale. Closing. Thanks! Please open a new PR if there is still a relevant change.

**Pull request closed without merging**